src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/CloseActionIT.kt

```diff
@@ -0,0 +1,52 @@
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.action
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.IndexStateManagementRestTestCase
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.Policy
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.State
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.action.CloseActionConfig
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.randomDefaultNotification
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.Locale
+
+class CloseActionIT : IndexStateManagementRestTestCase() {
+
+    fun `test basic`() {
+        val indexName = "${javaClass.simpleName.toLowerCase(Locale.getDefault())}_index"
+        val policyId = "${javaClass.simpleName.toLowerCase(Locale.getDefault())}_testPolicyName"
```

Comment:
NP: Saw that [declaration names with acronyms](https://kotlinlang.org/docs/reference/coding-conventions.html#choosing-good-names) are capitalized when they are less than three characters (ex. ```policyID```)

Reply:
nice. will do